### DISCUSSION TITLE
workloadClient.dialog.close() navigates to "/"

### DISCUSSION THREAD

**Discussion by WilcoHendriksen** [microsoft/Microsoft-Fabric-workload-development-sample#161] [General]
By default the workloadClient.dialog.close() navigates to "/" which seems a bit strange. The workload-sample-app doesn't have this route. If we put a console.log(route) in the "onNavigate" we can see its trying to navigate to "/". But if you defined it your router, it will navigate to that route. In newer versions of react-router this also gives an error if it does not exists. How should we handle this situation?

**Comment by shaym83** [comment]
No navigation is done automatically by the workloadClient.
When you call workloadClient.dialog.close(), the iFrame should be closed.
Can you place a breakpoint in onNavigate and check who is the caller?
